modules/persistent-storage-csi-resizing-procedure.adoc

```diff
@@ -0,0 +1,58 @@
+// Module included in the following assemblies:
+//
+// * storage/container_storage_interface/persistent-storage-csi-resizing.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="persistent-storage-csi-resizing-procedure_{context}"]
+= Resizing a CSI volume
+
+.Prerequisites
+* You are logged in to a running {product-title} cluster.
+* Your persistent volume claim (PVC) is created using a CSI driver that supports volume resizing.
+* Your storage back end is configured for dynamic provisioning.
+
+.Procedure
+
+To resize a volume:
```

Comment:
```suggestion

```
I think you can safely remove this line because it's clear from the title that the procedure is to resize a CSI volume.